.circleci/config.yml

```diff
@@ -11,14 +11,19 @@ parameters:
   system-contracts-path:
     type: string
     default: "compiled-system-contracts"
+  system-contracts-executor-image:
+    type: string
+      # The system contracts executor needs node 10 to function, this should
+      # only be changed when system-contracts-monorepo-version is changed.
+    default: "celohq/node10-gcloud:v3"
 executors:
   golang:
     docker:
       - image: circleci/golang:1.16
     working_directory: ~/repos/geth
-  node-v12:
+  system-contracts-executor:
     docker:
-      - image: us.gcr.io/celo-testnet/circleci-node12:1.0.0
+      - image: <<pipeline.parameters.system-contracts-executor-image>>
```

Comment:
ok, so default 10, unless we set this parameter in the ci config for the specific PR, right?

Reply:
node-v12(now system-contracts-executor) is only used for jobs.prepare-system-contracts

Reply:
@gastonponti that parameter is defined in the `.circleci/config.yml` at the top, so it's like a global variable of the config file.

https://github.com/celo-org/celo-blockchain/blob/e4277cccc7bff921cb7a8a28b9bce6511005bdbc/.circleci/config.yml#L14-L18

It's defined there rather than hardcoded so that it can be used in the cache key, so that if we change the image it invalidates the cache, and hence forces `prepare-system-contracts` to run fully